airbyte-cdk/python/airbyte_cdk/sources/declarative/auth/jwt.py

```diff
@@ -0,0 +1,102 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+from dataclasses import InitVar, dataclass
+from datetime import datetime
+from typing import Any, Mapping, Optional, Union
+
+import jwt
+from airbyte_cdk.sources.declarative.auth.declarative_authenticator import DeclarativeAuthenticator
+from airbyte_cdk.sources.declarative.interpolation.interpolated_mapping import InterpolatedMapping
+from airbyte_cdk.sources.declarative.interpolation.interpolated_string import InterpolatedString
+
+
+@dataclass
+class JwtAuthenticator(DeclarativeAuthenticator):
```

Comment:
I have no idea what our naming convention is — do we want acronyms to be all caps? I.e. `JWTAuthenticator` or `JwtAuthenticator`?

Reply:
I'm not attached to it but I find `JwtAuthenticator` a bit easier to read

Reply:
I followed `ApiTokenAuthenticator`'s lead.